example_dags/example_transform.py

```diff
@@ -69,6 +71,21 @@ def union_table_and_dataframe(input_table: Table, input_dataframe: pd.DataFrame)
 # [END transform_example_4]
 
 
+# [START transform_example_5]
+@aql.transform(
+    sql=str(pathlib.Path(CWD).parents[0])
+    + "/example_dags/demo_parse_directory/last_ten_animations.sql"
+)  # skipcq: PTC-W0049
+def last_ten_animations(input_table: Table):  # skipcq: PTC-W0049
+    """
+    Pass SQL file ending with .sql
+    """
```

Comment:
I think this is confusing for users, because the documentation is in the function.
```suggestion
    """
    Run a transformation using a sql file.
    """
```
WDYT?

Reply:
@feluelle makes sense.